src/frequenz/channels/util/_select.py

```diff
@@ -18,7 +18,7 @@
 
 
 class Selected(Generic[_T]):
-    """A result of a [`select`][frequenz.channels.util.select] iteration.
+    """A result of a [`select()`][frequenz.channels.util.select] iteration.
```

Comment:
So we always want to use `()` when referencing functions?

Reply:
In the rest of the file we always were referring to the select function as `select()`, so in this case I just wanted to have a consistent style.